pkg/kinflate/resource/configmap.go

```diff
@@ -27,8 +27,7 @@ import (
 	"k8s.io/kubectl/pkg/loader"
 )
 
-// NewFromConfigMap returns a Resource given a configmap metadata from manifest file.
-func NewFromConfigMap(cm manifest.ConfigMap, l loader.Loader) (*Resource, error) {
+func newFromConfigMap(cm manifest.ConfigMap, l loader.Loader) (*Resource, error) {
```

Comment:
You forgot to invert the order of `cm` and `l` here (it's not consistent with the public method).